frontends/bmd/src/pages/review_page.tsx

```diff
@@ -516,38 +528,46 @@ export function ReviewPage(): JSX.Element {
           }
         </VariableContentContainer>
       </Main>
-      <Sidebar
-        footer={
-          <React.Fragment>
-            <SettingsTextSize
-              userSettings={userSettings}
-              setUserSettings={setUserSettings}
-            />
-            <ElectionInfo
-              electionDefinition={electionDefinition}
-              ballotStyleId={ballotStyleId}
-              precinctSelection={singlePrecinctSelectionFor(precinctId)}
-              horizontal
-            />
-          </React.Fragment>
-        }
-      >
-        <SidebarSpacer />
-        <Prose>
-          <h2 aria-hidden>Review Votes</h2>
-          <p>Confirm your votes are correct.</p>
-          <p>
-            <LinkButton
-              large
-              primary
-              to={machineConfig.appMode.isPrint ? '/print' : '/save'}
-              id="next"
-            >
-              I’m Ready to <NoWrap>Print My Ballot</NoWrap>
-            </LinkButton>
-          </p>
-        </Prose>
-      </Sidebar>
+      {isPortrait ? (
+        <ButtonFooter>
+          {printMyBallotButton}
+          {/* <Button large onPress={() => {}}>
+            English
+          </Button> */}
+          <Button
+            large
+            onPress={() => setUserSettings({ showSettingsModal: true })}
+            aria-label="Change Settings"
+          >
+            Settings
+          </Button>
+        </ButtonFooter>
+      ) : (
+        <Sidebar
+          footer={
+            <React.Fragment>
+              <SettingsTextSize
+                userSettings={userSettings}
+                setUserSettings={setUserSettings}
+                sidebarWrapper
+              />
```

Comment:
We only have one setting right now, but as we add other settings, I'm thinking we'll want some sort of wrapper component containing all the settings to put in the sidebar in landscape mode. So we don't have different ordering or forget to add settings to a page. Thoughts?

Reply:
Not suggesting a change for this PR.

Reply:
Most-likely yes. However there are questions to answer here:
- How long will landscape orientation be supported?
- Do we want to modify the landscape orientation based upon what we've implemented for the portrait orientation?
- Will the landscape orientation use "Settings" and languages buttons similar to how the portrait orientation UI works?

Reply:
I don't know how long landscape will be supported, and I would like to _not_ support it for long, but I have a suspicion is will stick around or come back up.

Reply:
Agreed. FWIW, I tried to keep as little landscape-specific conditionals as possible. This one has been removed.